<?xml version="1.0" encoding="UTF-8"?>

<launch>

    <arg name="markerSize"      default="0.05"/>    <!-- in m -->
    <arg name="side"             default="left"/>
    <arg name="ref_frame"       default="base"/>  <!-- leave empty and the pose will be published wrt param parent_name -->


    <node pkg="laundryman" type="aruco_node" name="aruco_node" output="screen">
        <remap from="/camera_info" to="/xtion/rgb/camera_info" />
        <remap from="/image" to="/xtion/rgb/image_rect_color" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="reference_frame"    value="/base_footprint"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="/xtion_rgb_optical_frame"/>
        <param name="use_camera_info"       value="True"/>
    </node>

</launch>
